src/layouts/InstrumentLayout/components/Header/Header.tsx

```diff
@@ -50,7 +50,11 @@ const Header = ({ className = '' }: HeaderProps) => {
     setProfileDropdown(false);
   });
 
+  const dispatch = useDispatch<SequencerDispatch>();
   const settings = useSelector((state: RootState) => state.settings);
+  const { username, isLoading, error } = useSelector(
```

Comment:
на подушнить: errror и usLoading в компонентах может быть много (несколько запросов, с разных мест запрашиваем разные данные того, что загружается). Так что на моменте получения данных я советую их переименовывать:
```suggestion
  const { username, isLoading: isGetUsernameLoading, error: isGetUsernameError } = useSelector(
```